### ISSUE TITLE
Use of Responsive in the datatables instead of scrollX bar easier for mobile use 

### ISSUE THREAD

**Issue by Lymperis79** [nopSolutions/nopCommerce#4549]
nopCommerce version: 4.30

Steps to reproduce the problem:
Using the responsive addon in the datadables with minor changes ( just adding reference of the css and js files and replacing the scrollX: true with responsive: true in table.Definition page file.
The advantages is that the user can see the records table with out having to scroll down to find the scroll bar and return to the record especially in the mobile screen and he can see all the columns direct in a small area.

**Comment by AndreiMaz** [comment]
There was some reason why we haven't implemented it some time ago. We'll check it again

**Comment by DmitriyKulagin** [comment]
The table rendering strategy is very different when using these settings (scrollX and responsive). I tried both of these settings again on the latest versions of the libraries and this is what happened - "responsive" discards some of the columns to fit the table on the screen.

**scrollX** 
![scrollX](https://user-images.githubusercontent.com/27218280/143440506-b01695d6-d857-4fae-98c6-00ebd598343c.gif)

**responsive**
![responsive](https://user-images.githubusercontent.com/27218280/143441139-00bb1e56-d7a5-4077-ad69-9880d53e0359.gif)

Therefore, we have abandoned the use of the `responsive `function.
